### PR TITLE
[RFR] Added “Passport.js” entry

### PR DESCRIPTION
**Pull request by DtCarrot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +163,7 @@ If you are willing to contribute, open a pull request to complete, update or fil
 
 ### P
 
+* [Passport.js](/_glossary/PASSPORTJS.md): a simple authentication middleware for Node.js
```
Comment (KittyGiraudel) [inline README.md]: Any chance you could add the full stop? :)

File: _glossary/PASSPORTJS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Passport.js
```
Comment (KittyGiraudel) [inline _glossary/PASSPORTJS.md:6]: Do you mind adding a YAML Front Matter like this: https://raw.githubusercontent.com/HugoGiraudel/SJSJ/gh-pages/_glossary/AJAX.md?

**Comment by KittyGiraudel** [comment]
Awesome! :)

**Comment by DtCarrot** [comment]
Added fullstop and yaml front matter :)

**Comment by KittyGiraudel** [comment]
Super, thanks! :)

**Comment by KittyGiraudel** [comment]
http://jargon.js.org/_glossary/PASSPORTJS.md

**Pull request merged**